tools/preload.py

```diff
@@ -269,7 +269,10 @@ def fetch_webapp(app_url, directory=None):
                 manifest['package_path'],
                 os.path.join(app_dir, filename))
             metadata['manifestURL'] = url.geturl()
-            metadata['packageEtag'] = open_from_url(path).headers['etag']
+
+            package_url = open_from_url(path)
```

Comment:
I'll get rid of this change from this PR and move it to separate PR for the apppreload-toolkit repo - without this change we couldn't preload apps from the server that doesn't support ```Etag``` HTTP header (e.g GitHub Pages) - drawback is that we'll likely redownload package for every ```make raptor``` call since ```packageEtag``` will be empty.